ci/maketmpl/readme.go

```diff
@@ -0,0 +1,141 @@
+package maketmpl
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+)
+
+type readmeFile struct {
+	notebookFile string
+
+	md   []byte
+	imgs []*mdImage
+}
+
+func (f *readmeFile) writeInto(w io.Writer, imgOpts *writeImgOptions) error {
+	cursor := 0
+	for i, img := range f.imgs {
+		if cursor < img.start {
+			if _, err := w.Write(f.md[cursor:img.start]); err != nil {
```

Comment:
add comment what this mean `f.md[cursor:img.start]`?

Reply:
added comments that explains this function.

this function is copying the markdown with all the images rewritten.